pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/services/ReportPathGeneratorService.kt

```diff
@@ -27,12 +29,20 @@ class ReportPathGeneratorService(private val project: Project) {
         return Path.of("$projectBasePath/build/reports/pitest/test")
     }
 
+    private fun checkForDebugBuiltType(): Path {
+        var path = getReportPath()
+        if (Files.exists(Paths.get("$path/debug"))) {
```

Comment:
```suggestion
        if (Files.exists(Path.of("$path/debug"))) {
```
Sorry for the pickyness but lets keep it consistent :see_no_evil: